.github/workflows/builds.yml

```diff
@@ -62,29 +69,45 @@ jobs:
         BUILD_TYPE: ["Release", "Debug"]
         PLATFORM:
           - NAME: "HOST"
-            CONTAINER: "ghcr.io/acts-project/ubuntu2004:v43"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2404:68"
             OPTIONS: --preset eigen -DALGEBRA_PLUGINS_USE_SYSTEM_EIGEN3=TRUE
+            run_tests: true
           - NAME: "HOST"
-            CONTAINER: "ghcr.io/acts-project/ubuntu2004:v43"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2404:68"
             OPTIONS: --preset eigen
+            run_tests: true
           - NAME: "HOST"
-            CONTAINER: "ghcr.io/acts-project/ubuntu2004:v43"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2404:68"
             OPTIONS: -DALGEBRA_PLUGINS_INCLUDE_SMATRIX=TRUE
+            run_tests: true
           - NAME: "HOST"
-            CONTAINER: "ghcr.io/acts-project/ubuntu2004:v43"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2404:68"
             OPTIONS: --preset vc
+            run_tests: true
           - NAME: "HOST"
-            CONTAINER: "ghcr.io/acts-project/ubuntu2004:v43"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2404:68"
             OPTIONS: --preset vecmem
+            run_tests: true
           - NAME: "HOST"
-            CONTAINER: "ghcr.io/acts-project/ubuntu2004:v43"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2404:68"
             OPTIONS: --preset fastor
+            run_tests: true
           - NAME: "CUDA"
-            CONTAINER: "ghcr.io/acts-project/ubuntu2004_cuda:v43"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2404_cuda:68"
             OPTIONS: --preset cuda
-          - NAME: "SYCL"
-            CONTAINER: "ghcr.io/acts-project/ubuntu2004_oneapi:v43"
+            run_tests: false
+          - NAME: "SYCL HOST"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2404_oneapi:68"
+            OPTIONS: --preset sycl
+            run_tests: false
+          - NAME: "SYCL CUDA"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2404_cuda_oneapi:68"
+            OPTIONS: --preset sycl
+            run_tests: false
+          - NAME: "SYCL ROCM"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2404_rocm_oneapi:68"
             OPTIONS: --preset sycl
+            run_tests: false
```

Comment:
As @stephenswat pointed out as well, we should add something like an `ENV` variable, which the CI steps could use to set environment variables that are specific to the different jobs. This would allow moving the lines from `ci_setup.sh` here.